packages/cli/src/constructs/retry-strategy.ts

```diff
@@ -59,7 +59,7 @@ export class RetryStrategyBuilder {
     return {
       type,
       baseBackoffSeconds: options.baseBackoffSeconds ?? RetryStrategyBuilder.DEFAULT_BASE_BACKOFF_SECONDS,
-      maxAttempts: options.maxAttempts ?? RetryStrategyBuilder.DEFAULT_MAX_ATTEMPTS,
+      maxRetries: options.maxRetries ?? RetryStrategyBuilder.DEFAULT_MAX_ATTEMPTS,
```

Comment:
It's internal, but could you also rename `DEFAULT_MAX_ATTEMPTS`?

Reply:
Sure